crates/topos/src/components/tce/mod.rs

```diff
@@ -170,14 +171,18 @@ pub(crate) async fn handle_command(
 }
 
 pub fn print_node_info(config: &TceConfiguration) {
-    info!("TCE Node - version: {}", config.version);
+    tracing::warn!("TCE Node - version: {}", config.version);
 
     if let StorageConfiguration::RocksDB(Some(ref path)) = config.storage {
         info!("RocksDB at {:?}", path);
     }
 
-    info!("API gRPC endpoint reachable at {}", config.api_addr);
-    info!("Broadcast Parameters {:?}", config.tce_params);
+    tracing::warn!("API gRPC endpoint reachable at {}", config.api_addr);
+    info!(
+        "API GraphQL endpoint reachable at {}",
+        config.graphql_api_addr
+    );
+    tracing::warn!("Broadcast Parameters {:?}", config.tce_params);
```

Comment:
Import `tracing::warn` and use just `warn!()`, then it would be easy to replace some other tracing/log library if needed